jena-core/src/main/java/org/apache/jena/ext/xerces/impl/validation/ValidationState.java

```diff
@@ -86,11 +85,25 @@ public void setSymbolTable(SymbolTable sTable) {
         fSymbolTable = sTable;
     }
 
+    /**
```

Comment:
This methos could be removed. The maps shoudl be individually created on demand, as they are used in differenct methods and not always togehter.